Added snmp_user resource (#30)
netscaler/terraform-provider-netscalersdx: docs/resources/snmp_user.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "netscalersdx_snmp_user Resource - terraform-provider-netscalersdx"
subcategory: ""
description: |-
  Configuration for SNMP User resource.
---

# netscalersdx_snmp_user (Resource)

Configuration for SNMP User resource.

## Example Usage

```terraform
resource "netscalersdx_snmp_user" "tf_snmp_user" {
  name             = "tf_snmp_user"

>   security_level   = 2  # Please refer docs for these values      

Reviewer: ```suggestion
  security_level   = 2  # Please refer below for the valid values      
```
